ui_framework/doc_site/src/views/card/card_group.js

```diff
@@ -12,58 +11,82 @@ import {
 } from '../../../../components';
 
 export default () => {
+  /**
+   * These styles are just for demonstration purposes. It is recommended use
```

Comment:
Oops, had a typo in the proposed text :blush: better: `It is recommended to use...`